electron/menu/index.ts

```diff
@@ -0,0 +1,175 @@
+import { ConnectionStoreItem } from "@/lib/conn-manager-store";
+import { app, Menu, MenuItemConstructorOptions, shell } from "electron";
+import { isMac } from "../utils";
+import { createDatabaseWindow, windowMap } from "../window/create-database";
+import { OuterbaseApplication } from "../type";
+import { createWindow } from "../main";
+import { createConnectionWindow } from "../window/create-connection-window";
+import { OUTERBASE_GITHUB, OUTERBASE_WEBSITE } from "../constants";
+
+export function createMenu(
+  win: OuterbaseApplication["win"],
+  connections: ConnectionStoreItem[],
+) {
+  function handleClick() {
+    createWindow();
+  }
```

Comment:
We want to maintain only one main windows. We can have multiple database windows, but should only have one main window